docs/usage.rst

```diff
@@ -0,0 +1,209 @@
+.. Specify pandas format output in cells
+.. jupyter-execute::
+    :hide-code:
+    :hide-output:
+
+    import pandas as pd
+
+
+    def series_to_html(self):
+        df = self.to_frame()
+        df.columns = ['']
+        return df._repr_html_()
+
+
+    setattr(pd.Series, '_repr_html_', series_to_html)
+    pd.set_option('display.max_rows', 6)
+
+.. Specify version for storing and loading objects to YAML
+.. jupyter-execute::
+    :hide-code:
+
+    __version__ = '1.0.0'
+
+
+Usage
+=====
+
+The basic idea of :mod:`audinterface` is
+to provide easy and standardized interfaces
+to apply a machine learning model,
+or other digital signal processing algorithms
+to audio files.
+The only prerequisite is
+the algorithm provides a callable
+that takes at least the signal
+as a :class:`numpy.ndarray`
+and the sampling rate as input.
+
+The interface can then apply the algorithm
+on a list of files,
+a folder,
+or an index conform to the audformat_ database specification.
+Results are always returned containing a `segmented index`_.
+In the following we load three files from the emodb_ database
+and define a list of files,
+a folder,
+and an index.
+
+.. jupyter-execute::
+
+    import audb
+    import os
+
+    media = ['wav/03a01Fa.wav', 'wav/03a01Nc.wav', 'wav/03a01Wa.wav']
+    db = audb.load('emodb', version='1.2.0', media=media, verbose=False)
```

Comment:
I'd recommend we set `full_path=False` and then use `process_index(..., root=db.root)` to avoid showing the full path in the output.

Reply:
Good to know, maybe we should then also extend the docstring and be more precise there.
I thought when using `root=` it will also add this to the path in the returned index.

But if not, it is indeed much more elegant than my current solution.

Reply:
> it is indeed much more elegant than my current solution.

I just realized, this refers more to what I did in
https://github.com/audeering/audinterface/pull/48,
where I used `audformat.utils.map_file_path()` to remove the absolute path in the output.

Here, in the usage documentation I think it makes more sense to stay with `full_path=True` as this is the behavior the user would most likely need. The resulting dataframes will look ok in the HTML version of the docs, as the full path is usually shorter on the Github runner as it is when building locally.

Reply:
ok